_schemaV2.graphql

```diff
@@ -8436,7 +8436,7 @@ type Me implements Node {
     before: String
 
     # Exclude artworks that have been purchased on Artsy and automatically added to the collection.
-    excludePurchasedArtworks: Boolean
+    excludePurchasedArtworks: Boolean = false
```

Comment:
**issue:** do you know why this occurred although you didn't change it in the schema?

Reply:
This is coming from this: https://github.com/artsy/metaphysics/blob/faa9ec1113cd2133b5d2d65d1895845682b83dfd/src/schema/v2/me/myCollection.ts#L72

Reply:
weird, this should already be in the schema since it's not code that you added. not sure why it only got added now.